cetmix_tower_server/models/cx_tower_secret_mixin.py

```diff
@@ -0,0 +1,60 @@
+from odoo import api, fields, models
+
+
+class CxTowerSecretMixin(models.AbstractModel):
+    _name = "cx.tower.secret.mixin"
```

Comment:
Secrets and keys are implemented in the cx.tower.key model. So this model name should use the same name pattern.

Reply:
thanks, I'll fix it